specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2019-05-05-preview/AlertsManagement.json

```diff
@@ -337,8 +337,7 @@
           "200": {
             "description": "OK. Returns list of all smartGroups.",
             "schema": {
-              "$ref": "#/definitions/smartGroupsList",
-              "description": "List of smart groups in value property."
+              "$ref": "#/definitions/smartGroupsList"
```

Comment:
names for model definitions should be capitalized for consistency, I noticed you have some starting with lower case and some upper case in the "definitions" section.

Reply:
Done